jekyll/_components/tables.md

```diff
@@ -5,14 +5,159 @@ order: 9
+</div>
+</div>
+
 ## Styles
 
+Table rows alternate their background color, the subtle "zebra" effect assists in legibility:
+
+### Dark mode
+<div class="grid-2" markdown="1">
+![Network table in dark mode](../images/components/tables/table-grid-dark.png)
+
+<div markdown="1">
+Odd Row Background Color: Grey 80 `#2a2a2e`
+
+Even Row Background Color: `rgba(255,255,255,0.05)` (RGBA overlays the default Grey 80 `#2a2a2e`)
+
+Column Header Background Color: Grey 85 `#1b1b1d`
+
+Border/splitter Color: Grey 70 `#38383d`
+
+Text Color: Grey 40 `#b1b1b3`
+
+#### Selected row:
+
+Background Color: Blue 70 `#003eaa`
+
+Text Color: white `#ffffff`
+</div>
+</div>
+
+### Light mode
+<div class="grid-2" markdown="1">
+![Network table in light mode](../images/components/tables/table-grid-light.png)
+
+<div markdown="1">
+Odd Rows Background Color white `#ffffff`
+
+Even Rows Background Color: `rgba(0,0,0,0.05)` (RGBA overlays the default white `#ffffff`)
+
+Column Header Background Color: Grey 10 `#f9f9fa`
+
+Border/splitter Color: Grey 25 `#e0e0e2`
+
+Text Color: Grey 70 `#38383d`
+
+#### Selected row:
+
+Background Color: Blue 55 `#0074e8`
+
+Text Color: white `#ffffff`
+</div>
+</div>
+
+
+<div class="grid-2" markdown="1">
+![Table size and font properties](../images/components/tables/table-size-properties.svg)
+<div markdown="1">
+Font size: `11px`
+
+Row height: `24px`
+
+Line height: `24px`
+</div>
+</div>
+
 ## Behaviours
+
+### Sorting
+<div class="grid-2" markdown="1">
+![Table column headers](../images/components/tables/table-header-button.png)
+
+<div markdown="1">
+Tables should posses sortable column headers, clicking these buttons will toggle the sorting of the column they relate to. In the image, clicking the "Size" button header toggles the Size column from largest to smallest and from smallest to largest.
+
+#### Selected Column Header (Light mode):
+
+Background Color: Blue 55 `#0074e8`
+
+#### Selected Column Header (Dark mode):
+
+Background Color: Blue 70 `#003eaa`
+</div>
+</div>
+
+<div class="grid-2" markdown="1">
+![Table column tooltip](../images/components/tables/table-header-tooltip.png)
+
+<div markdown="1">
+If the table header is a clickable button to provide sorting controls, the tooltip should illustrate what the current sorting pattern is e.g. Sorted ascending, Sorted descending. 
+</div>
+</div>
+
+### Rich tooltips and previews
+
+<div class="grid-2" markdown="1">
+![Example of an image preview as tooltip](../images/components/tables/table-rich-tooltips.png)
+
+<div markdown="1">
+Table cell tooltips have the opportunity to provide extra information/context to the data being hovered over. For example, a domain name when hovered can additionally provide the IP address of the domain.
+
+When an image name is hovered over a [doorhanger](./doorhangers) should present a preview of the image with a caption containing the real pixel dimensions.
+
+</div>
+</div>
+
+## Future Improvements
+
```

Comment:
Another thing we could add here is [this bug](https://github.com/devtools-html/ux/issues/10) to possibly make the zebra effect a bit lighter (I see the classic zebra as a bit old fashioned due to the (imo) better workflow of hover color - but in particular, it seems that improving the hover color I mentioned in a comment could help usability.